lib/domain/entities/event.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:equatable/equatable.dart';
+
+class Event extends Equatable {
+  final String uid;
+  final String name;
+  final String cover;
+  final String status; // Offline/ Online Event
+  final String certificate;
```

Comment:
certificate can be later taken into separate model. to add few extra features like certificate type and validity, Certificate ID and issued by, so we can have good control of these details, can be shared without being mapped to event

Reply:
Oh @Harryalways317! @sreetejadusi is a bit weak at naming variables. Need to ask him to read @sampathbalivada's post on [`Naming conventions`](https://blog.sampath.dev/what-should-we-name-it). We actually wanted to add a base template of the certificate to each event, so that we could dynamically generate certificates and not store them anywhere with just an id.